components/omnibox/common/brave_omnibox_features.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_OMNIBOX_COMMON_BRAVE_OMNIBOX_FEATURES_H_
+#define BRAVE_COMPONENTS_OMNIBOX_COMMON_BRAVE_OMNIBOX_FEATURES_H_
+
+#include "base/feature_list.h"
+
+namespace omnibox {
+
+BASE_DECLARE_FEATURE(kOmniboxTabSwitchByDefault);
```

Comment:
pls add features into `components/omnibox/common/omnibox_features.h` via `chromium_src` override. This is a preferred way when you need to reference them from upstream files or in general use them closely with the upstream logic.

Reply:
Cool, that makes sense!